app/services/action_handlers/pull_request.rb

```diff
@@ -26,27 +26,16 @@ def closed
     end
 
     def review_request_removed
-      pr_data = @payload['pull_request']
-      reviewers_github_id = pr_data['requested_reviewers'].map { |reviewer| reviewer['id'] }
-      remove_review_requests(reviewers_github_id)
+      removed_reviewer = find_or_create_user(@payload['requested_reviewer'])
+      @entity.review_requests.find_by(reviewer: removed_reviewer, state: 'active')
```

Comment:
maybe the state wasn't active?

Reply:
We have active and removed. To check if it was reviewed just ask if it has al least one review.